src/lib/registration/downloader.rb

```diff
@@ -35,10 +35,21 @@ module Registration
   class Downloader
     extend Yast::Logger
 
+    # download a remote file via HTTP or HTTPS protocol, if maximum nuber or redirects
+    # is reached the download fails with RuntimeError exception
+    # @param file_url [String, URI] URL of the file to download
+    # @param insecure [Boolean] if true the SSL verification errors are ignored
+    # @return [String] the downloaded file
```

Comment:
"the contents of the downloaded file"

(an alternative would be to return the filename of the contents)